terraform/istio/install_istio.sh

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#!/bin/bash
+
+# Install Istio on the single cluster for Stackdriver Sandbox
+
+# Download Istio
+
+echo "### "
+echo "### Begin install istio control plane"
+echo "### "
+
+# Set vars for DIRs
+ISTIO_VERSION="${ISTIO_VERSION:-1.6.2}"
+
+# Set the working directory to our current directory (/sandbox/terraform)
+export WORK_DIR=`pwd`
```

Comment:
Scripts can be run from anywhere, we may not be in the directory you expect to be. It's a good practice to capture the location of the script itself, and then move into that directory to do your work.

There's some discussion about the best way to do that [here](https://stackoverflow.com/questions/4774054/reliable-way-for-a-bash-script-to-get-the-full-path-to-itself/4774063), but I believe this should work

```
SCRIPTPATH=$(dirname $(realpath $0))
cd $SCRIPTPATH
export WORKDIR = `pwd`
```

Reply:
Nice this makes the script a lot better. Changed :)